### DISCUSSION TITLE
CreateSLC error

### DISCUSSION THREAD

**Discussion by yagmurn** [alexisInSAR/EZ-InSAR#64] [Help]
Hi,

I have PAZ datasets. I selected the data directory that has unzipped data folders. I am choosing the mode as PAZ_SM and I am trying to create SLC list. But the list is empty and I have an error like below:

Index exceeds the number of array elements (0).

Error in createlistSLC (line 178)
        path_xml = [paramslc.pathSLC,'/',list_dir(i1).name,'/',path_xml(1).name];

Error in manageSLC (line 39)
        createlistSLC([],[],[],miesar_para);

Error in GUIMIESAR>@(src,evt,arg1,arg2)manageSLC(src,evt,'checking',figmiesar.UserData) (line 231)
btslccheck.ButtonPushedFcn = @(src,evt,arg1,arg2) manageSLC(src,evt,'checking',figmiesar.UserData);
 
Error using matlab.ui.control.internal.controller.ComponentController/executeUserCallback (line 382)
Error while evaluating Button PrivateButtonPushedFcn.

What should I do?

**Comment by alexisInSAR** [comment]
Hi [yagmurn](https://github.com/yagmurn), 

Can you describe your SLCs? The implementation of PAZ SM should worked but we have a few problems with the management of polarisations. 

Kind regards, 

Alexis

**Comment by yagmurn** [comment]
Hi @alexisInSAR,
Here is my SLC folder list ant its metadata.
![slc_list](https://github.com/alexisInSAR/EZ-InSAR/assets/74787162/b4798b8a-5bc9-4a27-a0f4-a12a4288815c)

![metadata2](https://github.com/alexisInSAR/EZ-InSAR/assets/74787162/32c06367-4a37-4313-8caf-42aab413274a)

**Comment by yagmurn** [comment]
Hi @alexisInSAR,

Is there any progress? Could you please give me any advices to process PAZ dataset if I will not solve this issue?

Best regards,

Nur

**Reply by alexisInSAR** [reply]
Hi, 

We are actively working on this implementation and it will be available in a few days. 

Kind regards, 

Alexis
